README.rst

```diff
@@ -61,11 +61,13 @@ matplotlib's communication channels include active mailing lists:
 * `Announcement  <https://mail.python.org/mailman/listinfo/matplotlib-announce>`_ mailing list: matplotlib-announce@python.org
 * `Development <https://mail.python.org/mailman/listinfo/matplotlib-devel>`_ mailing list: matplotlib-devel@python.org
 
-
 The first is a good starting point for general questions and discussions.
 
+.. image:: https://badges.gitter.im/Join%20Chat.svg
```

Comment:
I don't think we should repeat the image here; just have some text (like the alt text) pointing to the chat room.